### PR TITLE
C1 w1 review

### PR DESCRIPTION
**Pull request by e7dud7e**
Slightly renamed file names for ungraded labs
C1_W1_Lab_1: removed open in colab badge because it can run without GPU
C1_W1_Lab_2: removed open in colab badge because it can run without GPU
C1_W1_Lab_3: removed open in colab badge because it can run without GPU, reduced number of epochs from 2000 to 1000.  Changed wording from “we” to “you”.

C1_W1_Assignment
- For def final_model, added markdown to tell learn to list outputs in a specific order, else it won’t pass the unit test and also gets different results after training.  Order should be outputs=[wine_type,wine_quality].

- For model.compile, wrote markdown to avoid grading errors if they use alternative losses or metrics.  For example, sometimes they have to use a string and not the function or class.
- The ‘accuracy’ metric has to be lowercase even though the classes and functions that it’s based have an uppercase ‘A’.  Also, they can’t use tf.keras.losses.binary_crossentropy().
- ‘binary_crossentropy’ must be used and not the tf.keras.losses.binary_cross_entropy().
- They have to use ‘mse’ and not the function or class.
- They have to use tf.keras.metrics.RootMeanSquaredError() and not a string.

Note: If I set metrics: 'wine_type' : tf.keras.metrics.Accuracy() instead of ‘wine_type`=‘accuracy’, the unit test gives an error about the “loss” instead of the metric: wine quality loss function is incorrect. Please check implementation.
So I added markdown to tell the learner to check the other key value pairs even if the error message says that this line is wrong; because it might be another line that’s causing  the error.

Note that I only updated the C1W1_Assignment_Solution.ipynb and note the C1W1_Assignment.ipynb.  I recall Chris said that nbgrader can generate the student-facing notebook from the solution but I don’t know how to do that.

### CODE REVIEW

**Comment by e7dud7e** [comment]
@cfav-dev , I only modified the c1 w1 assignment solution and not the student-facing assignment.  If you have an automated way to generate this from the solution file, can you make this update before merge?  If not, let me know and I'll make the student-facing version too.

**Comment by e7dud7e** [comment]
For Lab 3, the reason I changed epochs from 2000 to 1000 is that if I run it in colab (with regular CPU), my browser window freezes and my computer starts to overheat before it can finish 2000 epochs.

**Comment by cfav-dev** [comment]
@e7dud7e thanks for the review! will look at the grader and modify to accept alternative solutions in `model.compile()`. I think it should accept the class, function, or string identifier.
